waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/DatabaseMappingImpl.java

```diff
@@ -66,7 +72,12 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import sun.security.krb5.RealmException;
```

Comment:
I think you need to optimise the imports here, look like leftovers from stuff you've tried and removed?

Reply:
yes, should be deleted now I think.